### DISCUSSION TITLE
I Can't See very well through the caves in the latest version

### DISCUSSION THREAD

**Discussion by RustyXD** [MinecraftFreecam/Freecam#238] [General]
When doing Freecam to Caves it doesn't Load Very well like the old versions, My Render and Simulation are good both in 10 just to reduce lag. But do anyone know how to fix it? Like I used to see all the caves and walls in 1.20. I'm not used to it tbh.

**Comment by MattSturgeon** [comment]
Are you using Iris shaders? You might be running into this issue: https://github.com/MinecraftFreecam/Freecam/issues/205

**Reply by RustyXD** [reply]
Only Iris To Support Shader's, But not using 1 just to reduce fps drop in my laptop

**Reply by MattSturgeon** [reply]
I'm not sure, but that issue may apply to Iris being installed rather than being enabled?

**Reply by RustyXD** [reply]
I'll try to remove it later just to test it again

**Reply by RustyXD** [reply]
Its Fixed now Thank You!!
